app/controllers/users_controller.rb

```diff
@@ -1,10 +1,8 @@
 class UsersController < ApplicationController
 
-  skip_before_filter :registration,
-                     only: [:ask_for_email, :edit, :update]
+  skip_before_filter :registration, only: [:ask_for_email]
```

Comment:
I agree, but you can get to the edit profile page now while you're still in the middle of registration (when you are not fully registered), so I think that if these are removed and you try to go to the profile page during registration you'll be taken back to registration.... probably should have written some specs (doh)